catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/FileSystemPersistenceProvider.java

```diff
@@ -215,7 +168,60 @@ public void clear() {
     }
   }
 
-  private File getMapStoreFile(String key) {
-    return new File(getMapStorePath() + key + SER);
+  Object loadFromPersistence(String key) {
```

Comment:
❓Should this be `private`? If it needs to be package-private for testing, we should add the `@VisibleForTesting` annotation.

Reply:
Yep, fixed